rustls/src/crypto/aws_lc_rs/ticketer.rs

```diff
@@ -21,11 +21,13 @@ use crate::server::ProducesTickets;
 pub struct Ticketer {}
 
 impl Ticketer {
-    /// Make the recommended Ticketer.  This produces tickets
+    /// Make the recommended `Ticketer`.  This produces tickets
     /// with a 12 hour life and randomly generated keys.
     ///
-    /// The encryption mechanism used is injected via TICKETER_AEAD;
-    /// it must take a 256-bit key and 96-bit nonce.
+    /// The `Ticketer` uses the [RFC 5077 §4] "Recommended Ticket Construction",
+    /// using AES 128 for encryption and HMAC-SHA256 for ciphertext authentication.
```

Comment:
Apologies, should've seen this in the previous review: let's use AES-256. The performance difference is not significant, but it feels unbalanced to use AES-128 to protect stronger keys even if the cryptographic strengths are so large that the difference is not meaningful in brute force settings. (Maybe a bit more meaningful in multi-key and post-quantum settings.)

Tried to comment all the places where there are mentions of AES-128 below, but there are also some in the commit message.

Reply:
Switched to AES 256 throughout (including commit message & PR desc)